packages/e2e/.mocharc.json

```diff
@@ -1,5 +1,5 @@
 {
   "require": "ts-node/register/files",
   "ignore": ["test/fixture-projects/**/*"],
-  "timeout": 60000
+  "timeout": 180000
```

Comment:
I increased this timeout as it fails very often when running under macos in the ci